### ISSUE TITLE
"Box obj cls total" loss gain is "nan" and the mAp is 0

### ISSUE THREAD

**Issue by Ganfan-Ren** [ultralytics/yolov5#3840]
## ❔Question
hi 

my Gpu is GTX 1650, Cuda version is 11.1, the torch version is 1.9.0

When I train coco128 datasets, it warn "Leaking Caffe2 thread-pool after fork. (function pthreadpool)", then I set the "batch-size" 1, this warning disappear. but the  "Box obj cls total" loss gain is "nan" and the mAp is 0. the detail of information as follows(result.txt)：

     0/299    0.489G       nan       nan       nan       nan        10       640         0         0         0         0       nan       nan       nan

**Comment by glenn-jocher** [comment]
@Ganfan-Ren train with default settings.

**Comment by Ganfan-Ren** [comment]
@glenn-jocher Thank you for your help. I do use the default settings. But now the problem has been solved. I updata my python  from v3.7 to v 3.8, because the python 3.8 is required by pytorch1.9.0.

**Comment by SunshineJZJ** [comment]
@Ganfan-Ren 
你好 我是GTX 1650 SUPER的显卡，python 3.8 、pytorch 1.7.1 cuda 11.0；
用6月份的代码可以正常训练；但是git clone 7月份的代码训练和你的问题一样，loss为nan，
请问您是如何修改的代码？

**Comment by abdulmoeedm2020** [comment]
I done the hole setup again, but result remain same
